File: app/src/main/java/me/chrislane/accudrop/db/converter/FallTypeConverter.java
package me.chrislane.accudrop.db.converter;

import android.arch.persistence.room.TypeConverter;

import me.chrislane.accudrop.db.FallType;

public class FallTypeConverter {

    @TypeConverter
    public static FallType toFallType(String falltype) {
        return FallType.valueOf(falltype);
    }

    @TypeConverter
    public static String toString(FallType falltype) {
        return falltype.name();
    }
}

File: app/src/main/java/me/chrislane/accudrop/db/converter/FallTypeConverter.kt
package me.chrislane.accudrop.db.converter

import androidx.room.TypeConverter

import me.chrislane.accudrop.db.FallType

class FallTypeConverter {

    @TypeConverter
    fun toFallType(fallType: String): FallType {
        return FallType.valueOf(fallType)
    }

    @TypeConverter
    fun toString(fallType: FallType): String {
        return fallType.name
    }
}
